spec/Overview.html

```diff
@@ -1391,9 +1414,10 @@ <h4>The encrypt method</h4>
                 <p>
                   If the following steps or referenced procedures say to
                   [= exception/throw =] an error,
```

Comment:
This is a rather confusing way of writing algorithms.

Reply:
Yeah, I agree. However, I also think it's a somewhat useful shorthand in this specific case because to get rid of this entirely, we would need to pass the promise and the realm to the internal operations, which would then each need to queue a task whenever it encounters an error or reaches a result.

Conceptually speaking, it's somewhat similar to saying
```js
return Promise.try(async function() {
  // remaining steps of this algorithm
});
```

But perhaps there's a better way of saying that?

Reply:
There is https://webidl.spec.whatwg.org/#an-exception-was-thrown but it's kinda weird to throw exceptions while in parallel as they are ECMAScript objects.